### DISCUSSION TITLE
SSL Certificate Error

### DISCUSSION THREAD

**Discussion by wcustodio** [mercadopago/sdk-js#80] [General]
I'm trying to config lib v2 in my Ionic/Cordova application, but I got this error when I used createCardToken function:

"MercadoPago.js - Your payment cannot be processed because the website contains credit card data and is not using a secure connection.SSL certificate is required to operate."

Is this a limitation for webview applications?

**Comment by lucmantovani** [comment]
Hello @wcustodio!

This error happens when you try to tokenize a card using production credentials and your environment doesn't have a SSL certificate, in any kind of application.

If you are still testing the integration, using test credentials should work.
If you are about to go to production, you should configure the SSL certificate.
